appservice/src/FileUtilities.ts

```diff
@@ -66,12 +66,17 @@ export async function zipDirectoryGitignore(folderPath: string, gitignoreName: s
     });
 }
 
+const commonGlobSettings: {} = {
+    dot: true, // Include paths starting with '.'
+    nodir: true, // required for symlinks https://github.com/archiverjs/node-archiver/issues/311#issuecomment-445924055
```

Comment:
@wwlorey might be worth double checking storage also supports symlinked folders since you were just messing with the glob settings